### PR TITLE
movieDetails upgrade

### PR DESCRIPTION
**Pull request by JacekWarzecha**

### CODE REVIEW

File: src/features/movieDetails/MovieDetails/Section/index.js
Comment (danielnowrot) [inline src/features/movieDetails/MovieDetails/Section/index.js:1]: less code - seems to be good ;)

**Review by danielnowrot — APPROVED** [review]
Good job!

**Review by danielnowrot — COMMENTED** [review]
Ok - this pull req cannot be merged?

**Review by przemek0000 — APPROVED** [review]
Fixed conflict. Good

**Pull request merged**